src/test/ui/consts/miri_unleashed/assoc_const_2.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0080]: erroneous constant used
+  --> $DIR/assoc_const_2.rs:27:13
+   |
+LL |     let y = <String as Bar<String>>::F; //~ ERROR erroneous constant
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^ referenced constant has errors
```

Comment:
"Has errors" is not very precise, why does it not show an actual miri error?

Reply:
because of `allow(const_err)` silencing the lint that reports the error on a constant